strings-csharp/ReadStringFromResourceFile/ReadStringFromResourceFile/Program.cs

```diff
@@ -1,12 +1,12 @@
 ﻿using ReadStringFromResourceFile;
 
-ResourcesManager rmEnglish = new ResourcesManager(@"ReadStringFromResourceFile.Resources.Texts.English");
+ResourcesManager rmEnglish = new(@"ReadStringFromResourceFile.Resources.Texts.English");
```

Comment:
`var` is the preferred way of writing these statements with the explicit constructor.